### ISSUE TITLE
atom's API has changed

### ISSUE THREAD

**Issue by stganser** [forivall/atom-hard-wrap#50]
hard-wrap stops working in atom 1.22.The following patch fixes the extension:

```
80c80,81
<           range = editor.rowRangeForParagraphAtBufferRow(range.getRows()[0])
---
>           range = editor.languageMode.
>             rowRangeForParagraphAtBufferRow(range.getRows()[0])
```

**Comment by mehaase** [comment]
Patch appears to be reversed, but other than it works for me.

**Comment by forivall** [comment]
fixed and released in 1.7.0
